.github/workflows/helm.yml

```diff
@@ -65,17 +76,10 @@ jobs:
         kubectl get pods
         kubectl logs $(kubectl get pods -l component=server -o jsonpath='{.items[0].metadata.name}')
 
-
-    - name: Generate SDK
-      run: |
-        pip3 install --user -r cvat-sdk/gen/requirements.txt
-        ./cvat-sdk/gen/generate.sh
-
-    - name: Install test requirements
-      run: |
-        pip3 install --user cvat-sdk/
-        pip3 install --user cvat-cli/
-        pip3 install --user -r tests/python/requirements.txt
+        if [[ max_tries -le 0 ]]; then
+          echo "CVAT failed to become ready in the expected time period"
+          exit 1
+        fi
```

Comment:
I like this, maybe it is worth adding similar lines for our main.yml as well https://github.com/opencv/cvat/blob/213bcf0b360d0e09fcfee1ff71ccf0d4e7a7b2b0/.github/workflows/main.yml#L313-L327